### DISCUSSION TITLE
are we back?

### DISCUSSION THREAD

**Discussion by pha1n0q** [arichornlover/uYouEnhanced#750] [General]
it looks like ari's github account is public and so is the repo now

**Comment by samuelawachie** [comment]
> it looks like ari's github account is public and so is the repo now
>
When the GitHub admins saw that our brain child YouTube was taken into the chamber of secrets, they saw it fit to summon @arichornlover back. One should hope that none of Lord Voldemorts old video tricks should make their way into innocent hands. The penalty for the one responsible could be severe.

**Comment by KingGamingYT** [comment]
escaped the shadowbanned hell

**Comment by arichornlover** [comment]
Wait.. what happened to all of my uYouEnhanced discussions 😭

**Reply by yodaluca23** [reply]
They all got closed, looks like...

**Reply by pha1n0q** [reply]
try setting the filter tab to all
idk why they're hidden like that though
![image](https://github.com/user-attachments/assets/f05e055c-a2f2-48c6-a535-95b46005713b)

**Reply by KingGamingYT** [reply]
I believe this happens automatically when a repo gets hidden, same thing happened to superminium

**Reply by arichornlover** [reply]
> I believe this happens automatically when a repo gets hidden, same thing happened to superminium

It also affected all other repos that had Discussions I’ve created. For example I had a discussion in “PoomSmart/YTABConfig” but it’s also “hidden” aswell. and filtering it to “All” in it will not work on those type of discussions. I tried?
